app/connectors/AssociationConnector.scala

```diff
@@ -92,25 +96,31 @@ class AssociationConnectorImpl @Inject()(httpClient: HttpClient,
 
   }
 
-  private def processResponse(response: HttpResponse, url: String) =
+  private def processResponse(acceptedInvitation: AcceptedInvitation,
+                              response: HttpResponse, url: String)(implicit request: RequestHeader, ec: ExecutionContext) = {
     if (response.status == OK) {
+      sendAcceptInvitationAuditEvent(acceptedInvitation, response.status, Some(response.json))
       Logger.info(s"POST of $url returned successfully")
       Right(())
     } else {
+      sendAcceptInvitationAuditEvent(acceptedInvitation, response.status, None)
```

Comment:
We might have a response here. The HTTP request has returned and a non-Ok response could contain some useful error message.

Reply:
Fixed